src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,48 @@
+package duke;
+
+import duke.tasks.TaskList;
+
+/**
+ * CLI Task manager based on Duke.
+ *
+ * @author bhcs
+ */
+public class Duke {
+
+    private final Ui ui;
+    private final Storage storage;
+    private final TaskList tasks;
+
+    /**
+     * Instantiates the Duke object and its required components.
+     */
+    public Duke() {
+        this.storage = new Storage();
+        this.tasks = storage.readFromDatabase();
+        Parser parser = new Parser(this.tasks);
+        ui = new Ui(parser);
+        storage.ui = ui;
+    }
+
+    protected void run() {
+        String logo =
+                " ____        _        \n"
+                        + "|  _ \\ _   _| | _____ \n"
+                        + "| | | | | | | |/ / _ \\\n"
+                        + "| |_| | |_| |   <  __/\n"
+                        + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo + '\n' + "What can I do for you?");
+        ui.monitor();
+        storage.writeToDatabase(tasks);
+    }
+
+    /**
+     * Begins main process of Duke.
+     *
+     * @param args java command line arguments.
+     */
+    public static void main(String[] args) {
+        new Duke().run();
+        System.out.println("a0");
```

Comment:
just curious, what's this for

Reply:
Oh haha oops! It was supposed to be removed, it was a debugging statement that I accidentally left in. I was testing out running `.run()` multiple times and deleted the extra calls but forgot this line. Thanks!